client/src/components/cards/CardsIndex.jsx

```diff
@@ -16,13 +30,29 @@ class CardsIndex extends Component {
   }
 
   render() {
+    const { classes } = this.props;
+
     return (
       <div>
         <h1> Astoria-arch Lean Coffee </h1>
         <h3>Add Your Topic</h3>
         <Form onSubmit={this.submit} />
         {this.props.cards.map(function(card) {
-          return(<p>{card.title}</p>)
+          return(
+            <Card className={classes.card}>
+              <CardContent>
+                {card.title}
+              </CardContent>
+              <CardActions>
+                <Button>
+                  EDIT
+                </Button>
+                <Button>
+                  VOTE
+                </Button>
+              </CardActions>
+            </Card>
+          )
         })}
```

Comment:
you can do .bind(this) to maintain the scope and you wont need to set that constant up top